app/models/school.rb

```diff
@@ -28,7 +28,7 @@ class School < ApplicationRecord
   has_many :induction_coordinator_profiles, through: :induction_coordinator_profiles_schools
   has_many :induction_coordinators, through: :induction_coordinator_profiles, source: :user
 
-  has_many :ecf_participant_profiles, through: :school_cohorts, source: :ecf_participant_profiles
+  has_many :ecf_participant_profiles, through: :school_cohorts, source: :ecf_participant_profiles, class_name: "ParticipantProfile::ECF"
```

Comment:
Why do we need this?

Reply:
join doesn't work otherwise `left_outer_joins(ecf_participant_profiles:`